Dockerfile

```diff
@@ -32,6 +32,8 @@ FROM node:lts-alpine
 
 WORKDIR /api
 
+RUN npm i -g nodemon ts-node typescript
```

Comment:
Install in image so that debugging session can be started from local environment